comp/core/tagger/README.md

```diff
@@ -62,31 +48,29 @@ cache. Cache invalidation is triggered by the collectors (or source) by either:
-| *Service*                               | *Tagger Entity*                                                    |
-|-----------------------------------------|--------------------------------------------------------------------|
-| workloadmeta.KindContainer              | `container_id://<sha>`                                             |
-| workloadmeta.KindContainerImageMetadata | `container_image_metadata://<sha>`                                 |
-| workloadmeta.KindGardenContainer        | `container_id://<sha>`                                             |
-| workloadmeta.KindKubernetesPod          | `kubernetes_pod_uid://<uid>`                                       |
-| workloadmeta.KindECSTask                | `ecs_task://<task-id>`                                             |
-| CloudFoundry LRP                        | `<processGuid>/<svcName>/<instanceGuid>`  or `<appGuid>/<svcName>` |
-| Container runtime or orchestrator       | (none)                                                             |
-| Kubernetes Endpoint                     | `kube_endpoint_uid://<namespace>/<name>/<ip>`                      |
-| Kubernetes Service                      | `kube_service://<namespace>/<name>`                                |
-| SNMP Config                             | config hash                                                        |
+| *Entity*                                | *ID*                               |
+|-----------------------------------------|------------------------------------|
+| workloadmeta.KindContainer              | `container_id://<sha>`             |
+| workloadmeta.KindContainerImageMetadata | `container_image_metadata://<sha>` |
+| workloadmeta.KindECSTask                | `ecs_task://<task-id>`             |
+| workloadmeta.KindHost                   | `host`                             |
+| workloadmeta.KindKubernetesDeployment   | `deployment://<namespace>/<name>`  |
+| workloadmeta.KindKubernetesNamespace    | `namespace://<name>`               |
+| workloadmeta.KindKubernetesNode         | `kubernetes_node_uid://<name>`     |
+| workloadmeta.KindKubernetesPod          | `kubernetes_pod_uid://<uid>`       |
+| workloadmeta.KindProcess                | `process://<pid>`                  |
 
 ## Tagger
 
-The Tagger handles the glue between **Collectors** and **TagStore** and the
-cache miss logic. If the tags from the **TagStore** are missing some sources,
-they will be manually queried in a block way, and the cache will be updated.
+The Tagger handles the glue between the workloadmeta collector and the
+**TagStore**.
 
 For convenience, the package creates a **defaultTagger** object that is used
 when calling the `tagger.Tag()` method.
 
-                   +-----------+
-                   | Collector |
-                   +---+-------+
+                   +--------------+
+                   | Workloadmeta |
```

Comment:
should this be `workloadmeta collector` instead? 

To me both seem ok, but `workloadmeta collector` is more informative.

WDYT?